CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 ## UNRELEASED
 
+BREAKING CHANGES:
+
+* ACLs: Remove `-expected-replicas`, `-release-name`, and `-server-label-selector` flags
+  in favor of the new required `-server-address` flag [[GH-238](https://github.com/hashicorp/consul-k8s/pull/238)].
+
+FEATURES:
+
+* ACLs: The `server-acl-init` can now run against Consul servers running outside of k8s [[GH-243](https://github.com/hashicorp/consul-k8s/pull/243)]:
+  * Add `-bootstrap-token-file` flag to provide your own bootstrap token. If set, the command will
```

Comment:
I would say "skip ACL bootstrapping". Because we still set server tokens and the word bootstrap is a bit overloaded.

Reply:
👍